### DISCUSSION TITLE
test car

### DISCUSSION THREAD

**Discussion by astrakid** [robinostlund/volkswagencarnet#265] [General]
hi @robinostlund 
i would offer you my car and my accoutn to do tests. i got my enyaq 2 weeks ago and i am not able to connect via api. the app "myskoda" is working fine.
if you are interested just contact me by mail andregronwald78@ gmail . com

regards,
andre

**Comment by robinostlund** [comment]
Hi @astrakid 
I have been thinking about looking into integrating skoda into this repository but have put that on hold as this component exists: https://github.com/skodaconnect/homeassistant-skodaconnect which i am currently using as i have a Skoda Enyaq Coupe RS. Have you tried that component?

**Reply by astrakid** [reply]
i tried it in iobroker, same result.
